### PR TITLE
Fixing SrRobotCommander

### PR DESCRIPTION
**Pull request by maxzieba**
## Proposed changes

User story: https://shadowrobot.atlassian.net/browse/SRC-6538
Fixing SrRobotCommander due to noetic migration. 

## Checklist

If the task is applicable to this pull request (see applicability criteria in brackets), make sure it is completed before checking the corresponding box. Otherwise, tick the box right away. Make sure that **ALL** boxes are checked **BEFORE** the PR is merged.

- [X] Manually tested that added code works as intended (if any functional/runnable code is added).
- [X] Added automated tests (if a new class is added (Python or C++), interface of that class must be unit tested).
- [X] Tested on real hardware (if the changed or added code is used with the real hardware).
- [X] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc.)

### CODE REVIEW

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py [hunk 1]
```diff
@@ -451,7 +451,9 @@ def plan_to_named_target(self, name, custom_start_state=None):
         else:
             self._move_group_commander.set_start_state(custom_start_state)
         if self.set_named_target(name):
-            self.__plan = self._move_group_commander.plan()
+            self.__plan = self._move_group_commander.plan()[1]
+        else:
+            rospy.logwarn("Named target does not exist")
```
Comment (mikramarc) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: maybe "failed to set to named target" is more accurate?

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py [hunk 2]
```diff
@@ -613,7 +615,7 @@ def run_named_trajectory_unsafe(self, trajectory, wait=False):
                             - pause_time -> time to wait at this wp
         """
         joint_trajectory = self.make_named_trajectory(trajectory)
-        if joint_trajectory is not None:
+        if type(joint_trajectory) == JointTrajectory:
```
Comment (mikramarc) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: we should add some info whan its not that.
so
if type(joint_trajectory) != JointTrajectory:
    bla bla
   return
self.run_joint_trajectory_unsafe(joint_trajectory, wait)

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py [hunk 3]
```diff
@@ -627,7 +629,7 @@ def run_named_trajectory(self, trajectory):
                           - pause_time -> time to wait at this wp
         """
         joint_trajectory = self.make_named_trajectory(trajectory)
-        if joint_trajectory is not None:
+        if type(joint_trajectory) == JointTrajectory:
```
Comment (mikramarc) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: same here

**Review by mikramarc — CHANGES_REQUESTED** [review]

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py (hunk 1 above)
Comment (maxzieba) [reply sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: Changed

**Review by mikramarc — APPROVED** [review]

**Review by georgiablanco — APPROVED** [review]

**Pull request merged**